documentation/ipi-install/ipi-install-upstream-master.adoc

```diff
@@ -16,6 +16,14 @@ Deployment Integration Team
 :sectlinks:
 :icons: font
 :product-title: OpenShift Container Platform
+:op-system-first: Red Hat Enterprise Linux CoreOS (RHCOS)
+:op-system: RHCOS
+:asb-name: OpenShift Ansible Broker
+:tsb-name: Template Service Broker
+:kebab: image:kebab.png[title="Options menu"]
+:rh-openstack-first: Red Hat OpenStack Platform (RHOSP)
+:rh-openstack: RHOSP
+:cloud-redhat-com: Red Hat OpenShift Cluster Manager
```

Comment:
Seems like we are using `op-system-first` and `op-system` but none of the others. I'm thinking not working add the extras.

Reply:
I just copied them to 'ensure' that once we get some other module using them, avoid getting 'blanks' in the docs should I remove them?